src/main/java/core/basesyntax/Countable.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Countable {

```

Comment:
The interface name 'Countable' is a verb, which is not recommended for class/interface names. Consider using a noun or a noun phrase that accurately describes the purpose of this interface, such as 'AreaCalculator' if its purpose is to calculate areas.